src/parser/expressions_parser.rs

```diff
@@ -341,9 +341,10 @@ fn parse_literal_number_with_modifier(
     let location = lexer.location();
     let token = lexer.slice_and_advance();
     let number_str = token.split('#').last().unwrap();
+    let number_str = number_str.replace("_", "");
 
     // again, the parsed number can be safely unwrapped.
-    let value = i64::from_str_radix(number_str, radix).unwrap();
+    let value = i64::from_str_radix(&number_str.as_str(), radix).unwrap();
```

Comment:
I would rather we don't panic here, this is no longer a safe unwrap as the empty string will crash.
Instead of unwrapping, go with either returning an empty statement on error with the "err" result being logged or doing the same check on line 344 where an empty string will log and return an empty_statement.

Reply:
Latest commit should tackle that. We should now be left with a stable verson that doesn't crash. Yet we can't detect multiple consecutive underscores. I'll open another issue for that, which we can solve after the logos folks have fixed the corresponding bug.

Reply:
I'll just add a few test and then it should be ready

Reply:
Done!